.github/workflows/builder.yaml

```diff
@@ -0,0 +1,53 @@
+name: Builder
+
+on:
+  pull_request:
+
+jobs:
+  check_for_membership:
+    runs-on: ubuntu-latest
+    name: Check membership of given user
+    outputs:
+      check-result: ${{ steps.composite.outputs.check-result }}
+
+    steps:
+      - name: Action for membership check
+        id: composite
+        uses: hazelcast/hazelcast-tpm/membership@main
+        with:
+          organization-name: 'hazelcast'
+          member-name: ${{ github.event.pull_request.head.repo.owner.login }}
+          token: ${{ secrets.GITHUB_TOKEN }}
+  default:
+    needs: [check_for_membership]
+    if: needs.check_for_membership.outputs.check-result == 'true'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Detect untrusted community PR
+        if: ${{ needs.check_for_membership.outputs.check-result == 'false' }}
```

Comment:
This can't ever happen because of the earlier `if: needs.check_for_membership.outputs.check-result == 'true'` check. I don't think you want both.

Reply:
Good catch
Fixed: [680f072949](https://github.com/hazelcast/hazelcast-code-samples/pull/674/commits/680f07294921c511527b914cb5646bc26bfdc9f0)